doc/stake-pool-operations/retire_stakepool.md

```diff
@@ -7,22 +7,10 @@ To retire a pool we need to:
 
 The deregistration certificate contains the _epoch_ in which we want to retire the pool. This epoch must be _after_ the current epoch and _not later than_ `eMax` epochs in the future, where `eMax` is a protocol parameter.
 
-So we first need to figure out the current epoch. The number of _slots per epoch_ is recorded in the genesis file, and we can get it with
+So we first need to figure out the current epoch. For this, the easiest way to query epoch would be by querying using EKG port (part of node config). To query the current epoch , you can use the command (where jq is a command line tool for parsing JSON, part of EPEL package on linux):
 
-    cat mainnet-shelley-genesis.json | grep epoch
-    > "epochLength": 21600,
-
-So one epoch lasts for 21600 slots. We get the current slot by querying the tip:
-
-    export CARDANO_NODE_SOCKET_PATH=relay-db/node-socket
-    cardano-cli shelley query tip --mainnet
-
-    > Tip (SlotNo {unSlotNo = 856232}) ...
-
-This gives us
-
-    expr 856232 / 21600
-    > 39
+    curl -s -H 'Accept: application/json' http://127.0.0.1:12788/ | jq -r .cardano.node.ChainDB.metrics.epoch.int.val
```

Comment:
The only thing missing here is that `hasEKG` needs to be enable in the config file and that the port number needs to match.

Add a note about `hasEKG` and this is good to go.

Reply:
Apologies, totally missed notification about this comment. Added now